.travis.yml

```diff
@@ -3,6 +3,7 @@ sudo: false
 script:
   - cargo build --verbose
   - cargo test --verbose
+  - cargo test --verbose --features pattern
   - cargo doc
```

Comment:
I agree. The docs also used to emphasize and use the `regex!` macro everywhere, but I removed almost all of it and de-emphasized use of `regex!` in the crate docs.